### PR TITLE
Doc + update Risk class

### PR DESCRIPTION
**Pull request by firstred**
| Questions | Answers |
| --- | --- |
| Branch? | `develop` |
| Description? | Upgrading the PHPDoc comments of this class, using the latest PS standards and also applying the PS standards to some of the code. Also getting the Franglais out of the comments, getting rid of things the Symfony code sniffer nags about, etc. It might have some additional `todo` tags, depending on whether the code was comprehensible or not. |
| Type? | improvement |
| Category? | CO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | N/A |
| How to test? | N/A |

Warning: this PR also has a code cleanup. Please check this first before merging!

### CODE REVIEW

File: classes/Risk.php [hunk 1]
```diff
@@ -63,16 +71,25 @@ public function getFields()
     public function getTranslationsFieldsChild()
     {
         $this->validateFieldsLang();
-        return $this->getTranslationsFields(array('name'));
+
+        return $this->getFieldsLang();
```
Comment (aleeks) [inline classes/Risk.php]: @julienbourdeau / @thierrymarianne / @maximebiloe / @mickaelandrieu
Another of the core team to confirm that it is ok?

Comment (aleeks) [inline classes/Risk.php]: @firstred I've no idea. This is the only class that has this feature...Waiting for another feedback ..

Comment (firstred) [inline classes/Risk.php]: I've just removed it. There are no usages anymore in the core and modules should have stopped using it since 1.5

**Comment by aleeks** [comment]
Thank you!

**Pull request merged**